src/functionalTest/groovy/com/autonomousapps/android/AbstractAndroidSpec.groovy

```diff
@@ -24,21 +24,23 @@ abstract class AbstractAndroidSpec extends AbstractFunctionalSpec {
   protected static final AGP_8_9 = AgpVersion.version('8.9.3')
   protected static final AGP_8_10 = AgpVersion.version('8.10.1')
   protected static final AGP_8_11 = AgpVersion.version('8.11.0')
+  protected static final AGP_8_12 = AgpVersion.version('8.12.0-alpha07')
```

Comment:
Android Studio Narwhal Feature Drop Canary 8 was released: https://androidstudio.googleblog.com/2025/07/android-studio-narwhal-feature-drop.html
But the corresponding AGP artefact is not yet available: https://maven.google.com/web/index.html#com.android.tools.build:gradle